index.html

```diff
@@ -2158,6 +2158,98 @@ <h3>Retrieve Verification Method</h3>
         </pre>
 
       </section>
+
+      <section class="normative">
+        <h3>Errors</h3>
+
+        <p>
+The algorithms described in this specification, as well as cryptographic
+suite specitications, throw specific types of errors. Implementers might find
+it useful to convey these errors to other libraries or software systems. This
+section provides specific URLs, descriptions, and error codes for the errors
+such that the ecosystem implementing technologies described by this
+specification might interoperate more effectively when errors occur.
+        </p>
+
+        <p>
+When exposing these errors through an HTTP interface, implementers SHOULD use
+[[RFC7807]] to encode the error data structure. If [[RFC7807]] is utilized:
+        </p>
+
+        <ul>
+          <li>
+The `type` value of the error object MUST be a URL that starts with the value
+`https://www.w3.org/ns/security/data-integrity#` and ends with the value in the
+section listed below.
```

Comment:
As I said in https://github.com/w3c/vc-data-integrity/issues/136#issuecomment-1658064597, I am not in favour of using that URL (and, at the minimum, there should be a note making it clear that the URL is not final).

All things considered, if we go along with this PR, then I would prefer to use `https://w3id.org/security#` instead.

Reply:
Fixed in 9abc28c8ae2259501081fc5db9fc760fd1232082.